README.md

```diff
@@ -18,17 +18,23 @@ Thanks!
 
 ## Install
 ---------
-- ` git clone https://github.com/johnomernik/qgrid`
+In one command:
+
+`pip install git+https://github.com/JohnOmernik/qgrid  https://github.com/JohnOmernik/qgrid/raw/main/qgrid2-1.1.3-py3-none-any.whl`
```

Comment:
What is the idea behind installing both
- `qgrid` from `main` (currently 1.3.1) and
- `qgrid2==1.1.3` from a wheel?

Reply:
`qgrid2` is qgrid javascript & png static files pre-canned, does not contain Python code if you look inside.